administrator-guides/custom-fields/README.md

```diff
@@ -1,60 +0,0 @@
-# Custom Fields
```

Comment:
Are these changes related to this PR? If not, please remove these changes. Thanks!